cmd/skaffold/app/cmd/init.go

```diff
@@ -52,6 +53,7 @@ func NewCmdInit() *cobra.Command {
 			f.BoolVar(&force, "force", false, "Force the generation of the Skaffold config")
 			f.StringVar(&composeFile, "compose-file", "", "Initialize from a docker-compose file")
 			f.StringArrayVarP(&cliArtifacts, "artifact", "a", nil, "'='-delimited Dockerfile/image pair, or JSON string, to generate build artifact\n(example: --artifact='{\"builder\":\"Docker\",\"payload\":{\"path\":\"/web/Dockerfile.web\"},\"image\":\"gcr.io/web-project/image\"}')")
+			f.StringArrayVarP(&cliKubectlManifests, "kubectl-manifest", "k", nil, "list of predefined kubectl manifests (overrides detection)")
```

Comment:
Any particular reason why its called kubectl-manifest instead of kubernetes-manifest?

Reply:
Because the deployer is called kubectl - there could be `kustomization-file` and `helm-releases`... But I agree that this might be confusing / looks janky off the bet. I think I am okay to change it to kubernetes-manifest.